concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -83,6 +83,7 @@ resources:
     source:
       uri: https://github.com/alphagov/paas-graphite-statsd-boshrelease.git
       tag_filter: {{cf_graphite_version}}
+      branch: 118272895_bosh20
 
```

Comment:
Good point.

Would you mind reviewing alphagov/paas-graphite-statsd-boshrelease#5, as you're testing the release with the new version of CF?

Do we need to specify `branch`if we're pinning the commit with `tag_filter` too?